wrapper/minecraft/src/main/kotlin/org/quiltmc/qkl/wrapper/minecraft/brigadier/util/ServerCommandUtil.kt

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qkl.wrapper.minecraft.brigadier.util
+
+import com.mojang.brigadier.context.CommandContext
+import net.minecraft.entity.Entity
+import net.minecraft.network.MessageType
+import net.minecraft.server.MinecraftServer
+import net.minecraft.server.command.ServerCommandSource
+import net.minecraft.server.network.ServerPlayerEntity
+import net.minecraft.server.world.ServerWorld
+import net.minecraft.text.Text
+import net.minecraft.util.registry.RegistryKey
+
+/**
+ * Player executing the command or
+ * `null` if not executed by a player.
+ *
+ * @see ServerCommandSource.method_44023
+ *
+ * @author Cypher121
+ */
+//TODO wait for mappings
+public val CommandContext<ServerCommandSource>.player: ServerPlayerEntity?
+    get() = source.method_44023()
+
+/**
+ * Entity executing the command or
+ * `null` if not executed by an entity.
+ *
+ * @see ServerCommandSource.entity
+ *
+ * @author Cypher121
+ */
+public val CommandContext<ServerCommandSource>.entity: Entity?
+    get() = source.entity
+
+/**
+ * World from which the command originates.
+ *
+ * @see ServerCommandSource.world
+ *
+ * @author Cypher121
+ */
+public val CommandContext<ServerCommandSource>.world: ServerWorld
+    get() = source.world
+
+/**
+ * Server the command is being executed on.
+ *
+ * @see ServerCommandSource.server
+ *
+ * @author Cypher121
+ */
+public val CommandContext<ServerCommandSource>.server: MinecraftServer
+    get() = source.server
+
+/**
+ * Responds to the player executing
+ * the command if one exists.
+ *
+ * @see ServerPlayerEntity.sendMessage
+ *
+ * @author Cypher121
+ */
+public fun CommandContext<ServerCommandSource>.respond(
+    message: Text,
+    messageType: RegistryKey<MessageType> = MessageType.SYSTEM
+) {
+    player?.sendMessage(message, messageType)
```

Comment:
This should probably be `source.sendFeedback` (iirc thats the name?)

Reply:
I may be blind

Reply:
replaced